packages/frontend/src/shared/ui/switches/ToggleSwitch.module.scss

```
@@ -0,0 +1,61 @@
+@use 'src/app/styles/core/colors' as *;
+@use 'src/app/styles/core/typography' as *;
+
+.container {
+  display: flex;
+  align-items: center;
+}
+
+.text-label-right {
+  @include body-5;
+  margin-left: 10px;
+}
+
+.text-label-left {
+  @include body-5;
+  margin-right: 10px;
+}
+
+.toggle {
+  position: relative;
+}
+
+.checkbox {
+  display: none;
```

Это скрытие инпута можно в глобальных стилях прописать и обозвать input-hidden
```suggestion
  position: absolute;
  z-index: -1;
  outline: none;
  opacity: 0;
```